replica/replica.go

```diff
@@ -918,6 +920,7 @@ func (r *Replica) createDisk(name string, userCreated bool, created string) erro
 		r.diskData[newSnapName] = r.diskData[oldHead]
 		r.diskData[newSnapName].UserCreated = userCreated
 		r.diskData[newSnapName].Created = created
+		r.diskData[newSnapName].RevisionCounter = r.GetRevisionCounter()
```

Comment:
Revision counter also needs to be updated while creating the newHeadDisk in CreateNewHead() func

Reply:
Not required as we want to keep only the available no of io's in the head, which will be zero when it's created. Later when replica will be restarted revision count will be updated with the information saved in revision.counter file.